### PR TITLE
Added new action to transitions

### PR DESCRIPTION
**Pull request by ar4s**
alternative to https://github.com/allegro/ralph/pull/1852

### CODE REVIEW

File: src/ralph/back_office/models.py [hunk 1]
```diff
@@ -195,3 +196,26 @@ def change_hostname(self, **kwargs):
             )
         }
     }
+
+    @transition_action
+    def change_user_and_owner(self, **kwargs):
+        UserModel = apps.get_model(*settings.AUTH_USER_MODEL.split('.'))
```
Comment (szok) [inline src/ralph/back_office/models.py]: Why do you not used : get_user_model() ?

File: src/ralph/lib/transitions/views.py [hunk 2]
```diff
@@ -30,6 +30,9 @@ def get_form_fields_from_actions(self):
         for action in self.actions:
             action_fields = getattr(action, 'form_fields', {})
             for name, options in action_fields.items():
+                condition = options.get('condition', lambda x: True)
+                if not condition(self.obj):
+                    continue
```
Comment (szok) [inline src/ralph/lib/transitions/views.py:35]: :+1: 

I think you have to added to documentation.

**Comment by ghost** [comment]
:ok:

**Comment by mkurek** [comment]
:ok:

**Pull request merged**